app/assets/stylesheets/map.css

```diff
@@ -51,3 +51,15 @@
 .leaflet-container .add-content-button {
   margin-right: 10px;
 }
+.leaflet-pane{
+  position: initial !important;
+}
+.leaflet-bar, .leaflet-top{
```

Comment:
This is awesome @itsgurmannatsohal :100: 
Maybe you could add the new property of `leaflet-top` to line 17, and have the remaining three "positions"  on one line since they are similar. What do think?
Thanks for working on  this @itsgurmannatsohal :heart:

Reply:
Sorry I was late. I was caught up in another work. I have made the changes but the tests are failing due to reasons unbeknownst to me, could you please clarify?
Thank you.

Reply:
No worries at all @itsgurmannatsohal , the css file is looking good. Though it looks like the last commit on plots2-main was added on your recent commit -  maybe a prob with the rebase?(files changed are now 3). Its no biggie as its the same as we have on main, but it would be cleaner to have only the changes you did on the issue. You could try removing them, but if it becomes a challenge just ping me and I am okay to merge it as is. Thanks again for working on this.

Reply:
Yes that is because I synced the plots2/main with my fork. I did that
1. To avoid any merge conflicts
2. Because the tests were failing when I didn't sync (so I synced and squashed the last two commits).

I can revert them if you'd prefer that.